gobblin-api/src/main/java/gobblin/configuration/State.java

```diff
@@ -244,7 +244,7 @@ public String getProp(String key, String def) {
    * @return value (the default value if the property is not set) associated with the key as a list of strings
    */
   public List<String> getPropAsList(String key, String def) {
-    return Splitter.on(",").trimResults().omitEmptyStrings().splitToList(getProperty(key, def));
+    return Splitter.on(",").trimResults().omitEmptyStrings().splitToList(getProp(key, def));
```

Comment:
You might want to define a constant for this `Splitter` to avoid creating a new instance every time this method is called.